flexget/components/ftp/sftp_client.py

```diff
@@ -0,0 +1,423 @@
+from typing import Callable, List, Optional
+import logging
+import os
+import posixpath
+import time
+from functools import partial
+from urllib.parse import quote, urljoin
+
+from flexget import plugin
+from flexget.entry import Entry
+
+# retry configuration constants
+CONNECT_TRIES: int = 3
+RETRY_INTERVAL: int = 15
```

Comment:
please add the measurement type into the variable name (i.e `RETRY_INTERVAL_SECONDS`)

Reply:
I think I'll add that as an optional config value, actually. Seems like something the user might want/need to adjust